doc/nrf/releases/release-notes-latest.rst

```diff
@@ -125,7 +125,51 @@ The current |NCS| release is based on Zephyr v2.5.99.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-* No changes yet
+* Drivers:
+
+  * Introduced the :c:macro:`DEVICE_DT_NAME` macro that returns a string name for a given devicetree node.
+  * Introduced the :c:func:`device_usable_check` function that determines whether a device is ready for use.
+
+  * Display:
+
+    * Added a driver and a generic shield definition for Sharp memory displays of the LS0XX type.
+
+  * Flash:
+
+    * Implemented workaround for nRF52 anomaly 242 in the nRF SoC flash driver.
+    * Added automatic selection of :option:`CONFIG_MPU_ALLOW_FLASH_WRITE` when the MPU is enabled for Arm based SoCs.
+
+  * GPIO:
+
+    Used the nrfx GPIOTE channel allocator in the nRF GPIO driver to properly track GPIOTE channel allocations made in other modules.
```

Comment:
```suggestion
    * Used the nrfx GPIOTE channel allocator in the nRF GPIO driver to properly track GPIOTE channel allocations made in other modules.
```

Reply:
Right. How could I miss that one? I think I'm getting old. :wink:

Reply:
Fixed.

I see that one more change sneaked in (the order of the mass storage Kconfig options) with that update. I hope it won't be a problem.